README.md

```diff
@@ -143,16 +150,18 @@ jobs:
           body: ${{ steps.calibre.outputs.markdown }}
 ```
 
-## Compressing images on a schedule
+## Compressing images on demand or on a schedule
 
 It is also possible to run image-actions on a recurring schedule. By using the `compressOnly` option, in conjunction with [`create-pull-request`](https://github.com/peter-evans/create-pull-request) action by [@peter-evans](https://github.com/peter-evans), a new Pull Request will be raised if there are optimised images in a repository.
 
 ```yml
-name: Compress images at 11pm and open a pull request
+# Compress images on demand (workflow), and at 11pm every Sunday (schedule).
+# Open a pull request if any images can be compressed
+name: Compress images
 on:
+  workflow:

```

Comment:
Does this mean it'd run after a merge to the main branch?

Reply:
Typo. Meant `workflow_dispatch` - fixed! See here: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/